adapter-s4j/src/main/resources/s4j.md

```diff
@@ -39,7 +51,7 @@ Other NB engine parameters are straight forward:
 
 # 3. NB S4J Driver Configuration Parameter File
 
-The S4J API has a list of configuration options that can be found here: https://docs.datastax.com/en/fast-pulsar-jms/docs/1.1/pulsar-jms-reference.html#_configuration_options.
+The S4J API has a list of configuration options that can be found here: https://docs.datastax.com/en/streaming/starlight-for-jms/3.2/reference/pulsar-jms-reference.html.
```

Comment:
#nitpick I'd change this to `latest` so it stays future-proofed.
```suggestion
The S4J API has a list of configuration options that can be found here: https://docs.datastax.com/en/streaming/starlight-for-jms/latest/reference/pulsar-jms-reference.html.
```

Reply:
Thanks @msmygit. Fixed